### PR TITLE
Result (Configuration) Deletion Fix 

### LINKED ISSUE #465: Analysis Parameters need to switch to new Configuration
Currently in ConfigInspector.js, when you delete a selected configuration, the selection switches over but the analysis parameters do not. Needs to be fixed so that the analysis parameters switch to the newly selected config as well. Additionally, please add a catch for when there is only 1 config and it gets deleted, so that a new default config is automatically made and added back

### PR DESCRIPTION
**Pull request by angelac2023**
If a result is deleted, then the user no longer has to click the updated selected result in order to see its analysis parameters. The analysis parameters update immediately following the result deletion.

Closes #465

### CODE REVIEW

**Comment by amgrubb** [comment]
Looks good, waiting on another team member to review.

**Review by YesugenB — APPROVED** [review]
Looks good! Good stuff. Just gotta fix the little conflict

**Review by amgrubb — APPROVED** [review]
Yup, agreed, just fix the conflict before merging.

**Pull request merged**
